### DISCUSSION TITLE
how to contact customer service supabase ?

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#29367] [Questions] [answered]
hello, I have a little problem to authenticate myself on my account supabase. Indeed the application that I used for my 2fa to delete my code. I went to the "Unable to sign in" section and filled out the form but wanted to know if this was the only way to contact them. Yes, I would like to solve this problem as soon as possible. If that is the only way then sorry for the inconvenience. How long does it take to deblock this about? I don’t know if I’m also in the right place for this topic.
Thank you for your help good day

**Comment by GaryAustin1** [comment, marked as answer]
If you could send the support request from your hosted dashboard that is the best way.  There is an email address of support@supabase.io.
Those are really the only methods.

**Comment by raf723** [comment]
Hi @thomasop, did you manage to regain access to your account? I'm having the same issue. I had to factory reset my phone and now I can't use Authy to gain access to my Supabase account anymore. 🙁
